corehq/apps/reports/standard/cases/basic.py

```diff
@@ -68,14 +70,31 @@ def _build_query(self):
             )
             query = query.NOT(case_es.owner(ids_to_exclude))
         else:  # Only show explicit matches
-            query = query.owner(self.case_owners)
+            selected_case_owners = self.case_owners
+            if selected_case_owners:
+                query = query.owner(selected_case_owners)
```

Comment:
Oh thats not good. Thanks for catching this. Will revert to old behaviour of not returning any result.